system-config/61-qubes-ignore-zvol-devices.rules

```diff
@@ -0,0 +1,3 @@
+# Determine if a zvol is a Qubes volume, then ignore it.
+KERNEL=="zd[0-9]*", SUBSYSTEM=="block", ACTION=="add|change", PROGRAM=="/usr/lib/udev/zvol_is_qubes_volume $devnode", ENV{QUBES_IS_PART_OF_QVM_POOL}="%c"
```

Comment:
Program spits out 1 or 0 depending on whether the volume is in fact tagged with the thing.